examples/Buttons/Button.md

```diff
@@ -72,6 +72,48 @@ import { VARIANT } from 'mark-one';
 </Button>
 ```
 
+
+The following is an example of a `form` prop for `button`
+
+```jsx
+import { useState } from 'react';
+
+const SimpleForm = (props) => {
+  
+  const [state, setState] = React.useState({ name: '' });
+  
+  const handleChange = ({ target }) => {
+    setState({ name: target.value });
+  };
+
+  const resetForm = () => {
+    setState({ name: '' });
+    document.getElementById("name").focus();
```

Comment:
This comment is made (somewhat) irrelevant by the on below regarding resetting form state, but I did want to just quickly write a note here to say that dropping to the native DOM API can cause odd issues between component re-renders.

The recommended way to do this tends to be to use a `useRef` hook and then pass the ref to the thing you wish to focus via the `forwardRef` prop. Then call `myRefName.current.focus()` in your handler to focus the element the ref is attached to.

You can see how to focus a `TextInput` with `useRef` [here](https://seas-computing.github.io/mark-one/#/Forms?id=textinput).